apps/web/src/views/Swap/AkkaSwap/hooks/types.ts

```diff
@@ -82,6 +82,7 @@ export interface RouteOperationsSeparated {
 }
 
 export enum TokenEnum {
-  NativeToken = 'BRISE',
+  BriseNativeToken = 'BRISE',
+  XdcNativeToken = 'XDC',
```

Comment:
There already is a mapping for the chains native token somewhere. That maps chainId to a token object, which has name and symbol